xrootd/xrdproto/sigver/sigver.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package sigver contains the structures describing sigver request.
+package sigver // import "go-hep.org/x/hep/xrootd/xrdproto/sigver"
+
+import (
+	"crypto/sha256"
+	"encoding/binary"
+
+	"go-hep.org/x/hep/xrootd/internal/xrdenc"
+	"go-hep.org/x/hep/xrootd/xrdproto/verifyw"
+	"go-hep.org/x/hep/xrootd/xrdproto/write"
+)
+
+// RequestID is the id of the request, it is sent as part of message.
+// See xrootd protocol specification for details: http://xrootd.org/doc/dev45/XRdv310.pdf, 2.3 Client Request Format.
+const RequestID uint16 = 3029
+
+type Flags uint8
+
+const (
+	NoData Flags = 1
+)
+
+// Request holds the sigver request parameters.
+type Request struct {
+	RequestId uint16
+	Version   byte
+	Flags     Flags
+	Seqno     int64
```

Comment:
could you add some documentation about `Version`, `Flags`, `Seqno` and `Crypto` ?
also, `Seqno` doesn't sound too Go-like.
what about `SeqID` ?